.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: Build
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
```

Comment:
Let's use v2 actions from day 1. They have extra benefits like faster checkout and setup process.